apps/dataset-browser/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Curious: can we put this file in the root of the repo (so that it can be used by all apps)?

Reply:
This file is automatically created after `npm run dev`. This is not customizable, and there is already an issue here: https://github.com/vercel/next.js/issues/42213 

For now, I can do nothing, but I will subscribe to the issue to know if there are any updates.